WooCommerce/Classes/ViewModels/Order Details/OrderDetailsDataSource.swift

```diff
@@ -205,9 +205,15 @@ final class OrderDetailsDataSource: NSObject {
 
     private let imageService: ImageService = ServiceLocator.imageService
 
-    init(order: Order, storageManager: StorageManagerType = ServiceLocator.storageManager) {
+    /// IPP Configuration loader
```

Comment:
nit: any reasons for including "loader" in the comment?
```suggestion
    /// IPP Configuration
```

Reply:
Copy-pasta!